components/mate/mate-screensaver/mate-screensaver.p5m

```diff
@@ -0,0 +1,158 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"), version 1.0.
+# You may only use this file in accordance with the terms of version
+# 1.0 of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source.  A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2016 Till Wegmueller
+#
+
+set name=pkg.fmri value=pkg:/$(COMPONENT_FMRI)@$(IPS_COMPONENT_VERSION),$(BUILD_VERSION)
+set name=pkg.summary value="$(COMPONENT_SUMMARY)"
+set name=info.classification value="$(COMPONENT_CLASSIFICATION)"
+set name=info.upstream-url value=$(COMPONENT_PROJECT_URL)
+set name=info.source-url value=$(COMPONENT_ARCHIVE_URL)
+set name=org.opensolaris.consolidation value=$(CONSOLIDATION)
+
+license $(COMPONENT_LICENSE_FILE) license='$(COMPONENT_LICENSE)'
+
+<transform file -> add pkg.depend.bypass-generate libGL\.so\.1>
+
+depend fmri=service/opengl/ogl-select type=require
+
+file path=etc/xdg/autostart/mate-screensaver.desktop
+file path=etc/xdg/menus/mate-screensavers.menu
+file path=usr/bin/mate-screensaver
+file path=usr/bin/mate-screensaver-command
+file path=usr/bin/mate-screensaver-preferences
+file path=usr/lib/pkgconfig/mate-screensaver.pc
+dir  path=usr/libexec/mate-screensaver
```

Comment:
We don't ship anything in usr/libexec, this should be moved to usr/lib and correct permissions should be set on files